src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -28,6 +30,8 @@ public sealed partial class TabBar : BaseTabBar, INotifyPropertyChanged
 
 		private bool _lockDropOperation = false;
 
+		private static int _gap = 6;
```

Comment:
Is this still needed?

Reply:
Yes, I would still leave a gap, most programmes use it. On a touchscreen it's very useful to have a space to catch the window. But I would probably use a gap (minimum size) of 40px.

Reply:
I meant that there should already be the correct size gap from the inset properties.

Reply:
Without a gap, the space is exactly the size of TitleBar.Inset(Right or Left), depending on the culture.

> ![image](https://github.com/user-attachments/assets/a01fec25-890b-438b-865c-07646fa091dd)

Green: TabBar.Inset
Red: minimum size `_gap`

Reply:
I'm seeing a different behavior on my side 
![image](https://github.com/user-attachments/assets/282a0683-a069-4dba-af77-649ec374ec41)

Reply:
it's the same, the size of TitleBar + _gap 🤔

Reply:
As you can see, the gap is already there, however if you follow the steps listed in #14598, the gap is not set correctly.

Reply:
> As you can see, the gap is usually there by default, however if you follow the steps listed in #14598, the gap is not set correctly.

For me so far the gap is always set correctly, I haven't found a problem yet

Reply:
If you mean this problem #14952 , I have already solved it in this PR